<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hao</class>
 <widget class="QDialog" name="hao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>chat</string>
  </property>
  <widget class="QTextBrowser" name="displayBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>441</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>350</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>266</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>传输文件</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
